webxr/resources/webxr_util.js

```diff
@@ -14,6 +14,10 @@
 var xr_debug = function(name, msg) {};
 
 function xr_promise_test(name, func, properties, glContextType, glContextProperties) {
+  if (glContextType === 'webgl2' && typeof WebGL2RenderingContext === 'undefined') {
+    // Do not run WebGL2 tests on platforms not supporting it.
+    return;
```

Comment:
This will make the tests simply pass, right? Would assert_implements be more appropriate to fail fast here?

Reply:
> This will make the tests simply pass, right?

It's more like a noop, the WebGL2 tests are not run.

> Would assert_implements be more appropriate to fail fast here?

One thing I wanted to fix is that a webgl2 test failure doesn't block other WebGL1 tests from running. Would `assert_implements` approach continue running the next tests when the assert fails?

Reply:
Aha, right, the early return is before `promise_test` is called. The result on wpt.fyi will be that some tests are missing without it being very obvious why. If you instead use `assert_implements` as the first thing inside `promise_test`, then those tests will fail fast. Unless the WebGL1 tests are triggered by something in those tests, those should still run. I suggest trying it and looking at the results at https://wpt.fyi/results/?label=pr_head&max-count=1&pr=28454 when they're updated.

Reply:
Thanks, I have tested the `assert_implements` solution and WebGL1 test can still be tested correctly. 

The PR is updated and all checks have passed now

Reply:
The flaky test didn't fail in the latest run, so if the code if ok it could also be landed by @klausw without the admin override